<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGIS2Mapbender</string>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../resources/icons/qgis2mapbender_ui_icon.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QRadioButton" name="credentialsPlainTextRadioButton">
             <property name="text">
              <string>Save credentials as plain text          </string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="userNameLabel">
             <property name="text">
              <string>User name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="passwordLabel">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QgsPasswordLineEdit" name="passwordLineEdit"/>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="authLabel">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="userNameLineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="credentialsAuthDbRadioButton">
             <property name="text">
              <string>Save credentials encrypted in authentication database</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="qgisServerPathLabel">
            <property name="text">
             <string>QGIS-Server URL*</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="qgisServerPathLineEdit"/>
            </item>
           </layout>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="internHLayout"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="mbBasisUrlLabel">
            <property name="text">
             <string>Mapbender basis URL*</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="mbBasisUrlLineEdit">
            <property name="inputMethodHints">
             <set>Qt::ImhUrlCharactersOnly</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="serviceNameLabel">
            <property name="text">
             <string>Server configuration name*                     </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="serverConfigNameLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Authentication</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="hLayout3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="testButton">
          <property name="text">
           <string>Test connection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="dialogButtonBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="locale">
           <locale language="English" country="Germany"/>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serverConfigNameLineEdit</tabstop>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
